ogr/abstract.py

```diff
@@ -581,7 +585,9 @@ def __str__(self) -> str:
             f"context='{self.context}', "
             f"uid='{self.uid}',"
             f"comment='{self.comment}',"
-            f"url='{self.url}')"
+            f"url='{self.url}'"
+            f"created='{self.created}'"
+            f"edited='{self.edited}')"
```

Comment:
I should point out, that GitLab does not have <edited> property in its status object. So this may be a problem.

Reply:
You can raise a `NotImplementedError` and add a comment with this info to the original GitHub issue.

Reply:
To make it possible, we can have properties for those values, so you can raise that error only when someone calls that.

Reply:
Okey, I adjusted the properties so it should be fine now.